UPGRADE-API-1.12.md

```diff
@@ -100,3 +100,12 @@ Here is how the response looks like:
 1. The 2nd parameter `localeCode` has been removed from `src/Sylius/Bundle/ApiBundle/Command/Cart/PickupCart.php` and now is set automatically by `src/Sylius/Bundle/ApiBundle/DataTransformer/LocaleCodeAwareInputCommandDataTransformer.php`.
 
 1. The responses of endpoints `/api/v2/admin/products` and `/api/v2/admin/products/{code}` have been changed in such a way that the field `defaultVariant` has been removed.
+
+1. To allow administrator reset his password, add in `config/packages/security.yaml` file the following entry
```

Comment:
I guess it has been already added in this file: https://github.com/Sylius/Sylius/pull/14149